js_modules/dagster-ui/packages/ui-core/src/graphql/types.ts

```diff
@@ -8436,10 +8436,10 @@ export const buildInstance = (
       overrides && overrides.hasOwnProperty('supportsConcurrencyLimits')
         ? overrides.supportsConcurrencyLimits!
         : false,
-    useAutomationPolicySensors:
-      overrides && overrides.hasOwnProperty('useAutomationPolicySensors')
-        ? overrides.useAutomationPolicySensors!
-        : true,
+    useAutoMaterializeSensors:
+      overrides && overrides.hasOwnProperty('useAutoMaterializeSensors')
+        ? overrides.useAutoMaterializeSensors!
+        : false,
```

Comment:
@salazarm do you have any ideas why this would change from true to false when all i changed was the name of the field? i am confused

Reply:
This is fine, it's just mock data